server/controllers/search/lib/entities_query_builder.js

```diff
@@ -70,7 +70,10 @@ const defaultEntitiesFields = userLang => {
   const fields = [
     'labels.*^2',
     'aliases.*^2',
-    'descriptions.*'
+    'descriptions.*',
+    // Boost titles and subtitles set in claims only
```

Comment:
why boost of 4 instead of 2 like labels ? wouldnt it favorise to much local collection entities compared to works entities for example?

Reply:
@jum-s that's a bit of a dirty hack to get the `search:entities: should return a local collection` test to fail less often >< The `collectionLabel` is generated using the `randomWords` function, which returns words that regularly happen to be for at least one of them in Wikidata collections labels, which then all get very high score, and push the local collection beyond the 100s rank... There seem to be an issue with the exact match on all words

Reply:
I removed the commit that added that boost (f215be88), and changed the test instead (35f44049)

Reply:
this is a fine tuning comment but maybe we can depreciate a bit de wikidata boost as you suggested some times ago (but not deleting it yet, there are still to many inv duplicates imho)

Reply:
done (10ce565d), let's see how it goes

Reply:
done (e419c049)